usr/share/rear/restore/CDM/default/400_restore_with_cdm.sh

```diff
@@ -0,0 +1,19 @@
+# 400_restore_with_cdm.sh
+#
+#
+
+LogPrint "Please start the restore process on the Rubrik (CDM) cluster."
+
+if is_true $CDM_NEW_AGENT_UUID; then
+  LogPrint ""
+  LogPrint "Register the appropriate IP address from this list with Rubrik (CDM):"
+  LogPrint "$( ip addr | grep inet | cut -d / -f 1 | grep -v 127.0.0.1 | grep -v ::1 )"
+  LogPrint ""
+fi
+LogPrint "Make sure all required data is restored to $TARGET_FS_ROOT ."
+LogPrint ""
+LogPrint "When the restore is finished type 'exit' to continue the recovery."
+LogPrint "Info: You can check the recovery process i.e. with the command 'df'."
+LogPrint ""
+
+rear_shell "Has the restore been completed and are you ready to continue the recovery?"
```

Comment:
@DamaniN 
from plain looking at your restore/CDM/default/400_restore_with_cdm.sh
I think this could perhaps be made more user friendly and more fail safe.

Currently it is one single attempt to do that restore process
and if that fails the user must redo the whole `rear recover`.

I think as an example how the user could retry things in case of failures
you may have a look at
https://github.com/rear/rear/blob/master/usr/share/rear/layout/recreate/default/200_run_layout_code.sh
how the user can run the disk layout recreation code (diskrestore.sh)
again and again until it succeeds or the user aborts `rear recover`.
Perhaps you could re-use that kind of code so that the user could
run the restore process on the Rubrik (CDM) cluster
again and again until it succeeds or the user aborts `rear recover`?

Reply:
In other examples where ReaR has to wait for the data to be restored from an external backup application, I don't see this type of retrying. The fact that the user is prompted to wait at the ReaR prompt and then `exit` to continue should be sufficient for them to perform any retries in the CDM software as needed. I'll look at providing a more automated workflow where ReaR can pull the restore job rather than have the user push a restore in the next release.